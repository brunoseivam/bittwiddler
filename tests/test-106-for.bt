main {
    var one_to_five:uint16[] = [1, 2, 3, 4, 5];

    for i, v in one_to_five {
        emit("one_to_five[{i}]: {v}\n");
    }
}
