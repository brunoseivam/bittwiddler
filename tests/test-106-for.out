one_to_five_u8[0]:  1
one_to_five_u8[1]:  2
one_to_five_u8[2]:  3
one_to_five_u8[3]:  4
one_to_five_u8[4]:  5
one_to_five_u16[0]: 1
one_to_five_u16[1]: 2
one_to_five_u16[2]: 3
one_to_five_u16[3]: 4
one_to_five_u16[4]: 5
one_to_five_u32[0]: 1
one_to_five_u32[1]: 2
one_to_five_u32[2]: 3
one_to_five_u32[3]: 4
one_to_five_u32[4]: 5
one_to_five_u64[0]: 1
one_to_five_u64[1]: 2
one_to_five_u64[2]: 3
one_to_five_u64[3]: 4
one_to_five_u64[4]: 5
one_to_five_i8[0]:  255
one_to_five_i8[1]:  254
one_to_five_i8[2]:  253
one_to_five_i8[3]:  252
one_to_five_i8[4]:  251
one_to_five_i16[0]: 65535
one_to_five_i16[1]: 65534
one_to_five_i16[2]: 65533
one_to_five_i16[3]: 65532
one_to_five_i16[4]: 65531
one_to_five_i32[0]: -1
one_to_five_i32[1]: -2
one_to_five_i32[2]: -3
one_to_five_i32[3]: -4
one_to_five_i32[4]: -5
one_to_five_i64[0]: -1
one_to_five_i64[1]: -2
one_to_five_i64[2]: -3
one_to_five_i64[3]: -4
one_to_five_i64[4]: -5
one_to_five_f32[0]: 0.000000
one_to_five_f32[1]: 0.000000
one_to_five_f32[2]: 0.000000
one_to_five_f32[3]: 0.000000
one_to_five_f32[4]: 0.000000
one_to_five_f64[0]: 1.000000
one_to_five_f64[1]: 2.000000
one_to_five_f64[2]: 3.000000
one_to_five_f64[3]: 4.000000
one_to_five_f64[4]: 5.000000
one_to_five_str[0]: 49
one_to_five_str[1]: 50
one_to_five_str[2]: 51
one_to_five_str[3]: 52
one_to_five_str[4]: 53
